### PR TITLE
Automatically timeout users with >3 automod actions in < 5 minutes

### PR DESCRIPTION
**Pull request by Kas-tle**
Adds a listener to automatically timeout users that accumulate 3 automod actions in under 5 minutes. This generally only occurs when an account has been compromised and is used for spam/scams.

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```
Comment (Chew) [inline gradle/wrapper/gradle-wrapper.properties:3]: This should be a separate PR, and should just upgrade all the other dependencies too

File: src/main/java/org/geysermc/discordbot/listeners/AutoModHandler.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+        event.getGuild().timeoutFor(member, 7, TimeUnit.DAYS);
+
+        User user = member.getUser();
+        user.openPrivateChannel().queue((channel) -> {
+            MessageEmbed embed = new EmbedBuilder()
+                .setTitle("You have been automatically timed out of GeyserMC!")
+                .addField("Reason", "Suspected account compromise", false)
+                .addField("Duration", "7 days", false)
+                .addField("Reccomended actions", "Change your Discord password, enable 2FA, and scan your computer for malware. See https://support.discord.com/hc/en-us/articles/24160905919511-My-Discord-Account-was-Hacked-or-Compromised for more info.", false)
```
Comment (Chew) [inline src/main/java/org/geysermc/discordbot/listeners/AutoModHandler.java:81]:
```suggestion
                .addField("Recommended Actions", "Change your Discord password, enable 2FA, and scan your computer for malware. See [Discord's article](https://support.discord.com/hc/en-us/articles/24160905919511-My-Discord-Account-was-Hacked-or-Compromised) for more info.", false)
```

File: src/main/java/org/geysermc/discordbot/listeners/AutoModHandler.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+        Member member = event.getGuild().getMemberById(userId);
+        if (member == null || member.isTimedOut()) return;
+
+        event.getGuild().timeoutFor(member, 7, TimeUnit.DAYS);
+
+        User user = member.getUser();
+        user.openPrivateChannel().queue((channel) -> {
+            MessageEmbed embed = new EmbedBuilder()
+                .setTitle("You have been automatically timed out of GeyserMC!")
```
Comment (Chew) [inline src/main/java/org/geysermc/discordbot/listeners/AutoModHandler.java:78]:
```suggestion
                .setTitle("You have been automatically timed out in GeyserMC!")
```

**Review by Chew — CHANGES_REQUESTED** [review]
Kinda neat :)

Nitpicks as usual

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (onebeastchris) [reply gradle/wrapper/gradle-wrapper.properties:3]: Given that we're often updating the gradle wrapper for e.g. Geyser in PRs that don't solely update dependencies, I see no reason to not do it here too

**Review by rtm516 — APPROVED** [review]
